docs/core-concepts/atomic-swap-htlc.md

```diff
@@ -0,0 +1,146 @@
+She shares the hash of that secret with Bob enabling him to fund on his side.
+The party in the cryptographic role of Alice has to fund and redeem first, because she has the actual secret.
+
+## Atomic Swaps using HTLCs in COMIT
+
+Prior to the actual swap execution there is usually a [Negotiation](negotiation.md) phase where the two parties agree on the ledgers, assets and amounts to be swapped.
+
+Assuming that Alice has ETH on Ethereum and Bob has BTC on Bitcoin, Alice wants Bob's BTC and Bob wants Alice's ETH, the execution of a swap would be like this:
+
+1. Alice generates a secret and hashes it. She sends to Bob:
+    1. Hash of the secret. 
+    1. Her refund address on Ethereum.
+    1. Her redeem address on Bitcoin.
+    1. Proposed expiry times for the HTLCs.
+1. Bob receives the swap proposed by Alice and sends back:
+    1. His refund address on Bitcoin.
+    1. His redeem address on Ethereum.
+1. Alice has to fund first, she deploys an Ethereum HTLC transferring the agreed amount of ETH into the contract.
+1. Bob monitors the Ethereum ledger and notices that Alice has deployed the HTLC funding the ETH. 
+He now sends a Bitcoin transaction into the Bitcoin network that is guarded by the script-hash of the Bitcoin HTLC (Bitcoin script).
+1. Alice monitors the Bitcoin ledger and notices that Bob has sent the funding transaction. She can now redeem the Bitcoin by spending the Bitcoin by revealing the secret.
+1. Bob is monitoring the Bitcoin ledger and notices that Alice has spent from the funding transaction by revealing the secret.
+He extracts the actual secret from Alice's redeem transaction and uses it to redeem on Ethereum. 
+
+The COMIT protocol acts as enabler for atomic swaps in applications.
+The [COMIT network daemon (cnd)](../comit-protocol/comit-protocol-stack.md#comit-network-daemon-cnd) mainly does two things:
+
+1. Monitor the ledgers by processing blocks supplied by blockchain nodes.
+1. Define the transaction details required for a swap and hand them over to an application built on top of COMIT.  The application is then responsible for sending the actual signed transactions into the respective blockchain network.
+
+<iframe 
+    src="https://docs.google.com/presentation/d/e/2PACX-1vTSToxZxKhOjfUWHVL5sjjjyGTdEXubjM3TpOpK-qR5Cjs7b6Tda9ZoX6n_NdM9iqgXBGOtPcPnjHAA/embed?start=false&loop=false&delayms=1000"
+    frameborder="0"
+    width="801"
+    height="480"
+    allowfullscreen="true"
+    mozallowfullscreen="true"
+    webkitallowfullscreen="true">
+</iframe>
+
+### HTLCs in COMIT
+
+In order to create HTLCs for the execution of an atomic swap, the two parties have to exchange certain parameters:
+
+1. Ledgers, assets and amounts to be swapped.
+1. Redeem and refund identity (address) of the respective parties.
+1. Expiry times to be used for the HTLCs.
+1. Hash of the secret (to be sent from the party in the cryptographic role of Alice to the party in the cryptographic role of Bob).
```

Comment:
Might be better to split the details into a separate sentence or removed entirely:

```
Hash of the secret. This hash is sent from the party in the cryptographic role of Alice to the party in the cryptographic role of Bob.
```